### ISSUE TITLE
Can't see the status and document type of a node when I haven't access to the settings section

### ISSUE THREAD

**Issue by vonbarnekowa** [umbraco/Umbraco-CMS#3174]
### Bug summary

When you haven't access to the settings section, you can't see the status and the document type of the selected node in the info tab.

### Steps to reproduce

- create a user account 
- restrict the access to the settings section for this specific user
- Go to the info tab of any node

### Expected result

Show the status and the document type.
![screenshot-1](https://user-images.githubusercontent.com/12753484/46537680-2a0c8e80-c8b2-11e8-9bef-d4fa97a9cb20.png)

### Actual result
The status and the document type aren't showed.
![screenshot-2](https://user-images.githubusercontent.com/12753484/46537682-2c6ee880-c8b2-11e8-9674-57bd0b2e452d.png)

**Comment by abjerner** [comment]
Hi @vonbarnekowa - thanks for creating this issue.

I think this may have been fixed in one of the recent releases of Umbraco. At least I can reproduce the issue in 7.12.3. I've tested this with a user that is part of the *Writers* group, and only has access to the content section.

In what version of Umbraco do you experience this?

**Comment by vonbarnekowa** [comment]
Thanks for the quick reply @abjerner!

I have the version 7.12.1 assembly: 1.0.6806.26190

**Comment by abjerner** [comment]
I remembered seeing a PR for this, but couldn't find it anywhere in the list of fixed bugs (which might be an error it self).

But knowing which file to look in, I found this commit:

https://github.com/umbraco/Umbraco-CMS/commit/ad3d27be76714dae57e51c1070f24691b3b7ae10#diff-a9d0c16524f07cd2b2d251b4cf2be2bb

It fixes the issue with the document type, and it seems to have been released in 7.12.2.

The status appears to have been fixed in this commit (which is also part of 7.12.2):

https://github.com/umbraco/Umbraco-CMS/commit/d4fff978144a1d6d88f32449f05ec59b5cced192#diff-9e0bba68d2f436d0ba79c2da547a21d8

So the solution would be to update your Umbraco installation. 7.12.2 and 7.12.3 are patch releases, so the update process should run smoothly.

7.12.3 also addresses an important [security issue in ClientDependency](https://umbraco.com/blog/security-advisory-patch-for-your-site-is-now-available/). This is kind of a must have if you haven't already updated ClientDependency manually,
